src/TMainConsole.cpp

```diff
@@ -176,7 +176,8 @@ void TMainConsole::toggleLogging(bool isMessageEnabled)
         if (mpHost->mLogFileNameFormat.isEmpty()) {
             if (mpHost->mLogFileName.isEmpty()) {
                 // If no log name is set, use the default placeholder
-                logFileName = tr("logfile", "Must be a valid default filename for a log-file and is used if the user does not enter any other value (Ensure all instances have the same translation {2 of 2}).");
+                //: Must be a valid default filename for a log-file and is used if the user does not enter any other value (Ensure all instances have the same translation {2 of 2}).
```

Comment:
Actually when I indexed these duplicated translatable `QString`s by adding a `{X of Y}` suffix in order to try and ensure that if one copy got changed other coders would realise that there were other `Y-1` cases to change I messed up and instead should have just said something like `{one of Y copies}` for ALL of them. The latter would mean that the Qt utilities would see an **identical** source in those Y cases whereas the former breaks that because `X` is different for each one and thus are NOT seen as identical texts... :man_facepalming:

Reply:
Ah, I see it now. What's the solution here? Should I just write `{one of 2 copies}`, for this example?

Reply:
That was the only one that I could find. I added them as `{one of two copies}`, as it seemed strange to me to add two numbers, one in arabic numerals and the other in latin script.